x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/api/rules/bulk_update_rules/route.ts

```diff
@@ -113,16 +104,13 @@ export const bulkUpdateRulesRoute = (
                   ruleId: payloadRule.id,
                 });
 
-                const rule = await updateRules({
-                  rulesClient,
-                  existingRule,
+                const rule = await rulesManagementClient.updateRule({
+                  ruleId: payloadRule.rule_id,
```

Comment:
Cannot we just take `rule_id` and `id` inside the `updateRule` method from the `payloadRule`? Same applies to `patchRule`

Reply:
Yes, we can 👍  fixing